tests/IntegrationTests/Client/GetBlockInfo.cs

```diff
@@ -12,6 +12,20 @@ public GetBlockInfo(ITestOutputHelper output) : base(output)
     {
     }
 
+    [Fact]
+    public async Task GivenBlockWithOneTransaction_WhenGetBlockInfo_ThenTransactionCountOne()
+    {
+        // Arrange
+        var blochHashString = this.GetString("blockHashOneTransaction");
```

Comment:
If you have some extra time, look into AutoFixture.
It can save a lot of boilerplate testing code when arranging dummy values, mocks and objects.

Reply:
We should also add coverage for when a block contains more than one transaction & verify the expected behavior when the hash doesn't point to valid block. Meaning Info cannot be found.

Reply:
Please don't introduce `AutoFixture`, it hides the test data which most of the time will be crucial for the test.

Reply:
Removed since none of the other SDK’s have test like this.